docs/moderne-cli/README.md

```diff
@@ -1609,8 +1709,7 @@ git diff
 Finally, you can commit the changes to all the repositories at once with the following command:
 
 ```shell
-mod commit $HOME/workshop -m "Migrate to spring boot 3" --recipe-run 20230905113712-JvUVv
+mod commit $HOME/workshop -m "Migrate to spring boot 3" --last-recipe-run
 ```
 
-Notice that you can also use `mod checkout` to create a branch for each repository and then use `mod commit` 
-to commit the changes. This might be useful if you want to create a pull request for each repository.
+If you'd rather make a branch for each repository and make changes in that, you can use the `mod checkout` command before running `mod commit` to commit the changes. This might be useful if you want to create a pull request for each repository.
```

Comment:
I rephrased this a bit as I thought this was somewhat confusing. I'm not sure if I phrased things right, though -- so would appreciate a check on this.